doc/tutorials/using_vxworks_as_uos.rst

```diff
@@ -0,0 +1,137 @@
+.. _using_vxworks_as_uos:
+
+Using VxWorks as User OS
+########################
+
+This tutorial describes how to run VxWorks as the User OS on the ACRN hypervisor. We are using Kaby Lake-based
+NUC (model NUC7i5DNHE) in this tutorial. Other :ref:`ACRN supported platforms <hardware>` should work as well.
+
+Introduction to VxWorks
+***********************
+
+VxWorks is a real-time operating system (RTOS) that can be used in embedded systems. It can be customized easily
+and run on most processors designed for distributed computing.
```

Comment:
Perhaps we could also use Windriver's own description of VxWorks here, indicating we're quoting them, e.g.:
```
Quoting Wind River's product page: VxWorks®, the industry’s leading real-time operating system, delivers all the performance, determinism, reliability, safety, and security capabilities you need to meet the highest standards for running the computing systems of the most important critical infrastructure.
```

Reply:
Sure. Will do.